scripts/split-val-entry.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+xxd -l 32 -ps -c 40 /dev/urandom > /tmp/nitro-val.jwt
+echo launching validation
+/usr/local/bin/nitro-val --file-logging.file nitro-val.log --auth.addr 127.0.0.10 --auth.origins 127.0.0.1 --auth.jwtsecret /tmp/nitro-val.jwt --auth.port 2000 &
```

Comment:
Maybe we should select a less common port to avoid conflicts here. Hardcoding a fixed number probably makes sense, but using something random instead of 2000 might be better.